### PR TITLE
PROGRAMMES-6829 - "Subscribe in Sounds" button

### PR DESCRIPTION
**Pull request by Bobkidbob**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6829

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: templates/podcast/podcast.html.twig [hunk 1]
```diff
@@ -45,35 +45,14 @@
...
-                                                    {{ gelicon('basics', 'no', 'gelicon--gamma') }}
-                                                </span>
-                                                {{ tr('subscribe_to_podcast')}}
-                                            </label>
-                                        </h3>
-                                        <ul class="list-unstyled popup__list">
-                                            {{ self.subscribeLink('pcast', entity, 'itunes', 'iTunes / iOS', 'progs-icons', 'apple')  }}
-                                            {{ self.subscribeLink('https', entity, 'android', 'Android / Google Play', 'progs-icons', 'android')  }}
-                                            {{ self.subscribeLink('pcast', entity, 'windows', 'Windows Phone', 'progs-icons', 'windows') }}
-                                            {{ self.subscribeLink('https', entity, 'rss', 'RSS', 'basics', 'rss')  }}
```
Comment (BjornTwachtmann) [inline templates/podcast/podcast.html.twig]: Now that the subscribe popup has been removed, you can remove the subscribeLink macro and javascripts block from the end of this template

File: src/Controller/Podcast/PodcastController.php [hunk 2]
```diff
@@ -82,6 +84,21 @@ public function __invoke(
 
         $schema = $this->getSchema($structuredDataHelper, $programme, $downloadableVersions, $coreEntity);
 
+        switch ($coreEntity->getType()) {
+            case 'brand':
```
Comment (BjornTwachtmann) [inline src/Controller/Podcast/PodcastController.php]: Hmm, I just realised there's a problem here. There are plenty of series TLECs that we need to provide a subscribe link for, e.g.  series b015sqc7
https://www.bbc.co.uk/programmes/b015sqc7/episodes/downloads
https://www.bbc.co.uk/sounds/play/brand:b015sqc7.

It's probably easiest just to make the brand case the default.

File: src/Ds2013/Css/PageSpecfic/_podcast.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    line-height: 1;
+}
+
+.podcast--subscribe-link {
+    padding: 12px;
+    margin-bottom: 8px;
+}
+
+a.podcast--subscribe-link,
+a.podcast--rss-link {
```
Comment (BjornTwachtmann) [inline src/Ds2013/Css/PageSpecfic/_podcast.scss]: Why change the default link styling here?

File: templates/podcast/podcast.html.twig [hunk 4]
```diff
@@ -45,35 +45,14 @@
...
-                                </div>
-                            </div>
-                        </div>
+                        <nav class="island">
+                            <a href="{{ soundsSubscribeUrl }}" class="popup__button popup__button--subscribe br-linkinvert podcast--subscribe-link">
+                                {{ gelicon('basics', 'external-link', 'podcast--subscribe-icon') }}
+                                {{ tr('subscribe_sounds') }}
+                            </a>
+                            <br />
+                            <a href="https://podcasts.files.bbci.co.uk/{{ entity.getPid() }}.rss" class="podcast--rss-link">RSS</a>
```
Comment (BjornTwachtmann) [inline templates/podcast/podcast.html.twig]:
```suggestion
                            <a href="{{ url('podcast_download_rss', {'pid' : entity.getPid()}) }}" class="podcast--rss-link">RSS</a>
```
Not sure why that wasn't being used in here before, but we might as well be consistent

**Pull request merged**